bindings/postgres/postgres.go

```diff
@@ -0,0 +1,176 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package postgres
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+	"time"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/dapr/pkg/logger"
+	"github.com/pkg/errors"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+// List of operations.
+const (
+	ExecOperation  bindings.OperationKind = "exec"
+	QueryOperation bindings.OperationKind = "query"
+	CloseOperation bindings.OperationKind = "close"
+
+	ConnectionURLKey = "url"
+	CommandSQLKey    = "sql"
+
+	CommandTimeoutDefault = 5
```

Comment:
should this be a configurable property?

Reply:
Actually not needed at all, the connection string can have all these properties so already configurable.